static_report/src/components/ComparisonReport/CRTableList/CRTableListColumnList.tsx

```diff
@@ -15,52 +17,55 @@ export function CRTableListColumnList({
   baseTableDatum: TableSchema;
   targetTableDatum: TableSchema;
 }) {
-  const mergedAssertionColumns = Object.keys(
-    baseTableDatum.piperider_assertion_result?.columns || {},
-  ).map((colName) => {
-    const mergedBaseColAssertions = [
-      ...(baseTableDatum.piperider_assertion_result?.columns?.[colName] || []),
-      ...(baseTableDatum.dbt_assertion_result?.columns?.[colName] || []),
-    ];
-
-    const mergedTargetColAssertions = [
-      ...(targetTableDatum.piperider_assertion_result?.columns?.[colName] ||
-        []),
-      ...(targetTableDatum.dbt_assertion_result?.columns?.[colName] || []),
-    ];
+  const transformedData = transformAsNestedBaseTargetRecord<
+    TableSchema['columns'],
+    ColumnSchema
+  >(baseTableDatum.columns, targetTableDatum.columns);
 
+  const columns = Object.keys(targetTableDatum.columns).map((colName) => {
     const { icon: colIcon } = getIconForColumnType(
       targetTableDatum.columns[colName],
     );
 
-    const transformedData = transformAsNestedBaseTargetRecord<
-      TableSchema['columns'],
-      ColumnSchema
-    >(baseTableDatum.columns, targetTableDatum.columns);
+    const mergedBaseColAssertions = [
+      ...get(
```

Comment:
hmm, i'm not sure why we need `get`, doesn't this escape typescript type inference?

Reply:
Yes, type inference is broken. `get` just for convenience is unnecessary.